dev/helpers/mssql/read_required_migrations.sql

```diff
@@ -0,0 +1,20 @@
+CREATE OR ALTER PROCEDURE ReadRequiredMigrations
+    @migrationsFile NVARCHAR(MAX)
+AS
+BEGIN
+    SET NOCOUNT ON;
+
+    CREATE TABLE #InputMigrations (Filename NVARCHAR(MAX))
+
+    DECLARE @bulkInsertSql NVARCHAR(4000) = 'BULK INSERT #InputMigrations FROM ''' + @migrationsFile + ''' WITH (FIELDTERMINATOR = '';'', ROWTERMINATOR = '';'')';
+    EXEC(@bulkInsertSql);
+
+    -- Select migrations that do not appear in the [dbo].[Migrations] table
```

Comment:
Super ⛏️ The table name is actually lowercase right? But the comment capitalizes.